engine/api/workflow/run_workflow_test.go

```diff
@@ -218,16 +220,16 @@ func TestManualRun2(t *testing.T) {
 	})
 	test.NoError(t, err)
 
-	_, _, err = workflow.ManualRun(nil, db, db, cache, proj, w1, &sdk.WorkflowNodeRunManual{
+	_, _, err = workflow.ManualRun(context.TODO(), db, cache, proj, w1, &sdk.WorkflowNodeRunManual{
 		User: *u,
 	}, nil)
 	test.NoError(t, err)
 
-	_, _, err = workflow.ManualRun(nil, db, db, cache, proj, w1, &sdk.WorkflowNodeRunManual{User: *u}, nil)
+	_, _, err = workflow.ManualRun(context.TODO(), db, cache, proj, w1, &sdk.WorkflowNodeRunManual{User: *u}, nil)
 	test.NoError(t, err)
 
 	//TestprocessWorkflowRun
-	_, _, err = workflow.ManualRunFromNode(nil, db, db, cache, proj, w1, 1, &sdk.WorkflowNodeRunManual{User: *u}, w1.RootID)
+	_, _, err = workflow.ManualRunFromNode(nil, db, cache, proj, w1, 1, &sdk.WorkflowNodeRunManual{User: *u}, w1.RootID)
```

Comment:
do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use